node-fs-readfile/cat.js

```diff
@@ -0,0 +1,9 @@
+process.argv.splice(0, 2);
+const allFiles = process.argv;
+const fs = require('fs');
+for (let i = 0; i < allFiles.length; i++) {
+  fs.readFile(allFiles[i], 'utf-8', (err, data) => {
+    if (err) throw err;
+    console.log(data);
+  });
+}
```

Comment:
Good attempt! This will print all of the files the user inputs, however the order of the output won't be guaranteed. Also we don't want to directly make changes to `process.argv` by calling `splice` on it.